src/elements/element.line.js

```diff
@@ -166,6 +166,11 @@ function fastPathSegment(ctx, line, segment, params) {
 	}
 }
 
+/**
+ * @priavte
```

Comment:
new typo :-)

Reply:
usually we put `@private` last

Reply:
Yeah, but `@priavte` deserves a top position! :)
Fixed